self-core-impl/src/main/java/com/selfxdsd/core/GithubIssueComments.java

```diff
@@ -28,24 +34,49 @@
 
     /**
      * Ctor.
+     *
      * @param issueUri Comments Issue URI.
      * @param resources Github's JSON Resources.
      */
     public GithubIssueComments(final URI issueUri,
                                final JsonResources resources) {
-        this.commentsUri = issueUri.resolve("comments");
+        final String issueUriStr = issueUri.toString();
+        String slash = "/";
+        if(issueUriStr.endsWith("/")){
+            slash = "";
+        }
+        this.commentsUri = URI.create(issueUriStr + slash + "comments/");
         this.resources = resources;
     }
 
     @Override
-    public Comment post(final String body) {
-        //POST /repos/:owner/:repo/issues/:issue_number/comments
-        throw new UnsupportedOperationException("Not implemented yet");
+    public Comment post(final String body, final String accessToken) {
```

Comment:
@criske This method should not have the accessToken as parameter.

Initially, I wanted the token to come from the ``Github`` parent: passed to ``GithubIssues`` in constructor, then passed to ``GithubIssue`` and finally to ``GithubComments``. 

But now, I see it would  be better if ``JdkHttp`` encapsulated the token as well. I will do that in #248. 

I suggest you leave this method unimplemented for now and write a ``todo`` for implementing it later.

Reply:
Yeah, didn't have access to accessToken, so I had to update the api, or go refactoring all over the project. It's way much better if  `JdkHttp` encapsulate the token.

Wouldn't be better if, after reverting to initial `post`, I just pass an empty string to `JsonResources#post`? The code is still correct plus tests don't rely on token anyway.

Reply:
@criske yes, that's also ok :)